main.go

```diff
@@ -344,8 +367,9 @@ func main() {
 	s.Use(middleware.Recover())
 
 	var handler = &admissionWebhookServer{
-		config: conf,
-		logger: logger.Named("admissionWebhookServer"),
+		config:    conf,
+		logger:    logger.Named("admissionWebhookServer"),
+		clientset: getClientSet(),
 	}
```

Comment:
```suggestion
	}
	_ , clientset, err := k8s.NewVersionedClient()
	if err != nil {
		logger.Fatal(err.Error())	
	}
```

Reply:
@denis-tingaikin I've added fixes as you proposed. Please, take a look.